payu/archive_scripts/build_intake_ds.py

```diff
@@ -0,0 +1,57 @@
+#!python3
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+# modules:
+# use:
+#     - /g/data/hh5/public/modules
+# load:
+#     - conda/analysis
+
+from access_nri_intake.source import builders
+import os
+
+from pathlib import Path
+import sys
+
+path_root = Path(__file__).parents[2]
+sys.path.append(str(path_root))
+
+from scripts_common import get_provenance_metadata, md5sum
+
+from ruamel.yaml import YAML
+from ruamel.yaml.comments import CommentedMap
+
+METADATA_FILENAME = "metadata.yaml"
+UUID_FIELD = "experiment_uuid"
+
+builder = builders.AccessOm3Builder(path="archive/")
+
+builder.build()
+
+# Log invalid assets
+builder.invalid_assets
+
+# Get experiment uuid
```

Comment:
I could probably import a fucntion to do this from a payu submodule? That probably makes more sense ?

Reply:
Something like (untested):

```python
from payu.metadata import Metadata

metadata = Metadata("archive/")
uuid = metadata.uuid
```

You'll still need to handle the case where `UUID_FIELD` does not exist. Currently you're setting `uuid` to `None` in that case, which will fail at line 54.